README.md

```diff
@@ -55,234 +20,123 @@ After you're finished please remove all the comments and instructions!
 - [🤝 Contributing](#contributing)
 - [⭐️ Show your support](#support)
 - [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ (OPTIONAL)](#faq)
+- [❓ FAQ](#faq)
 - [📝 License](#license)
 
-<!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
 
-> Describe your project in 1 or 2 sentences.
+# 📖 Capstone 1 The LOTR Lovers-Conference <a name="about-project"></a>
+
 
-**[your_project__name]** is a...
 
-## 🛠 Built With <a name="built-with"></a>
+**[Capstone 1 The LOTR Lovers-Conference is my second project and I prepared a conference project]**
 
+## 🛠 Built With<a name="built-with"></a>
+ <ul>
+    <li>HTML</li>
+    <li>CSS</li>
+    <li>JS</li>
+    <li>Git</li>
+    <li>GitHub</li>
+  </ul>
 ### Tech Stack <a name="tech-stack"></a>
 
-> Describe the tech stack and include only the relevant sections that apply to your project.
 
-<details>
-  <summary>Client</summary>
-  <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
-  </ul>
-</details>
 
-<details>
-  <summary>Server</summary>
-  <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
-  </ul>
-</details>
 
-<details>
-<summary>Database</summary>
-  <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
-  </ul>
-</details>
 
-<!-- Features -->
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+
+- **[Creating linters and using them]**
+- **[Creating JS, html and css files]**
+- **[Opening a pull request]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
-
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+URL:https://otmantr.github.io/Capstone-1-The-LOTR-Lovers-Conference/
```

Comment:
[OPTIONAL] Kindly use the correct markdown to show this link. Using github markdown makes your code look so much better.To learn more about github markdown, see [link](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links)

Reply:
@Nyame-Wolf Thank you for your suggestion.